core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -263,10 +263,31 @@ class PlaintextConsumerTest extends BaseConsumerTest {
   }
 
   @Test
-  def testSeek() {
+  def testSeekUnCompressedMessages() {
     val consumer = this.consumers(0)
     val totalRecords = 50L
-    sendRecords(totalRecords.toInt)
+    sendRecords(totalRecords.toInt, tp)
+    consumer.assign(List(tp).asJava)
```

Comment:
Rather than duplicate this code I think there are a couple of ways you can cover both cases in the same method fairly cleanly.